src/devices/video/topcat.h

```diff
@@ -29,8 +36,12 @@ class topcat_device : public device_t
 	DECLARE_READ16_MEMBER(ctrl_r);
 	DECLARE_WRITE16_MEMBER(ctrl_w);
 
+	WRITE_LINE_MEMBER(vblank_w);
 	void topcat_mem(address_map &map);
 
+	bool plane_enabled();
+
+	devcb_write_line m_int_write_func;
```

Comment:
This really shouldn't be public - if you're relying on subdevices being able to access it directly, there will likely be a problem if they can't coordinate with each other.